src/network/networkd-route.c

```diff
@@ -1723,12 +1723,16 @@ int manager_rtnl_process_route(sd_netlink *rtnl, sd_netlink_message *message, Ma
                 return 0;
         }
 
-        r = sd_rtnl_message_route_get_table(message, &table);
+        r = sd_netlink_message_read_u32(message, RTA_TABLE, &tmp->table);
+        if (r == -ENODATA) {
+                r = sd_rtnl_message_route_get_table(message, &table);
```

Comment:
In the future, please add some comment in the commit message why a change like this is done… That makes it much easier to review.

Reply:
Sorry. Several comment is added now.